src/rpc/net.cpp

```diff
@@ -169,6 +171,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         obj.pushKV("relaytxes", stats.fRelayTxes);
         obj.pushKV("lastsend", stats.nLastSend);
         obj.pushKV("lastrecv", stats.nLastRecv);
+        obj.pushKV("last_transaction", stats.nLastTXTime);
```

Comment:
nit, I don't have an alternative but `last_transaction` suggests it returns the actual transaction, not a timestamp.

Reply:
> nit, I don't have an alternative but `last_transaction` suggests it returns the actual transaction, not a timestamp.

I considered adding `_time` to the names, but we already have timestamp fields both with and without "time" (`conntime`, `lastsend`, `lastreck`), so I went for the shorter variants unless everyone prefers `last_transaction_time` and `last_block_time` instead.

Reply:
I'm fine with either, although I think `last_block_time` could still be confusing, since one interpretation could be the timestamp in the block.